backend/prisma/seed/config.seed.ts

```diff
@@ -116,6 +116,12 @@ const configVariables: ConfigVariables = {
       defaultValue: "false",
       secret: false,
     },
+    allowUnauthenticatedCertificates: {
```

Comment:
I would rename `allowUnauthenticatedCertificates` to `allowUnauthorizedCertificates`

Reply:
Yeah, my bad, I even wanted to call it that, but I didn't :D